src/electron/views/device-connect-view/components/android-setup/prompt-choose-device-step.tsx

```diff
@@ -26,33 +50,42 @@ export const PromptChooseDeviceStep = NamedFC<CommonAndroidSetupStepProps>(
             console.log(`androidSetupActionCreator.rescan()`);
         };
 
-        const items = [
+        // Available devices will be retrieved from store in future feature work
+        const devices: DeviceMetadata[] = [
             {
-                value: 'Phone 1',
+                description: 'Phone 1',
+                isEmulator: true,
             },
             {
-                value: 'Phone 2',
+                description: 'Phone 2',
+                isEmulator: false,
             },
             {
-                value: 'Phone 3',
+                description: 'Phone 3',
+                isEmulator: true,
             },
         ];
 
+        const items = devices.map(m => ({ metadata: m }));
+
         const layoutProps: AndroidSetupStepLayoutProps = {
             headerText: 'Choose which device to use',
             children: (
                 <>
-                    <p>2 Android devices or emulators connected</p>
+                    <p>{devices.length} Android devices or emulators connected</p>
                     <DefaultButton text="Rescan" onClick={onRescanButton} />
                     <DetailsList
+                        setKey={'devices'}
```

Comment:
bc of https://github.com/microsoft/fluentui/issues/7817 and https://github.com/microsoft/fluentui/issues/3366